presto-docs/src/main/sphinx/admin/tuning.rst

```diff
@@ -26,8 +26,8 @@ These configuration options may require tuning in specific situations:
   require redistributing both tables using a hash of the join key. This can
   be slower (sometimes substantially) than broadcast joins, but allows much
   larger joins. Broadcast joins require that the tables on the right side of
-  the join fit in memory on each machine, whereas distributed joins only
-  need to fit in distributed memory across all machines. This can also be
+  the join fit in memory on each machine, whereas with distributed joins the right
```

Comment:
I think this should say "with distributed joins the tables on the right side have to fit in distributed memory"